### DISCUSSION TITLE
Pictures of school ID instantly getting rejected (name issue)

### DISCUSSION THREAD

**Discussion by HEJOK254** [community/community#124161] [GitHub Education]
### Hi everyone,

I have been trying to apply for the Student Developer Pack, and my applications are instantly getting rejected (at least when I'm trying to use my school ID).

What I think the problem could be, is the fact that my full name on the ID is handwritten, and the person who wrote it seems to have made a slight mistake and one letter looks like "l" instead of "k", which I'm guessing is making some kind of automatic verification bot fail the application.

I have quadruple checked if my billing information and the full name on my profile matches.

This is the error I'm getting;
![image](https://github.com/community/community/assets/90698026/f8c7a52b-33dd-4dcf-b1d4-b2b3abc36d8c)

**Side note:** I also have a school email, however it doesn't seem to get recognized, possibly because it doesn't end with .edu

**Comment by linker34** [comment]
What I think the problem could be, is the fact that my full name on the ID is handwritten, and the person who wrote it seems to have made a slight mistake and one letter looks like "l" instead of "k", which I'm guessing is making some kind of automatic verification bot fail the application.
